e2e/test/iothub/DeviceTokenRefreshE2ETests.cs

```diff
@@ -115,14 +115,11 @@ public async Task IotHubDeviceClient_CreateFromConnectionString_TokenIsRefreshed
             using var deviceDisconnected = new SemaphoreSlim(0);
 
             TestDevice testDevice = await TestDevice.GetTestDeviceAsync(Logger, DevicePrefix).ConfigureAwait(false);
+            var auth = new DeviceAuthenticationWithConnectionString(testDevice.ConnectionString, sasTokenTimeToLive, sasTokenRenewalBuffer);
```

Comment:
We now have a specific IAuthenticationMethod that accepts connection string and the sas ttl settings. The regular CreateFromConnectionString no longer accepts configurable sas ttl.